components/global_header/left_controls/history_buttons/history_buttons.tsx

```diff
@@ -40,13 +43,46 @@ const HistoryButtons = (): JSX.Element => {
     const goBack = () => {
         trackEvent('ui', 'ui_history_back');
         browserHistory.goBack();
+        window.postMessage(
+            {
+                type: 'history-button',
```

Comment:
I take it there's no way for the desktop app to detect navigation to trigger these messages for us?

Also, just to check, does this way of doing it still work if you hit the back button and then click on another channel to navigate without going forward in history?

Reply:
Correct, I checked and Electron still can't detect changes in the `browserHistory` module, so we have to do it manually.

And yes it still works, we also fire the event on the `browserHistory.push` event so that clicking on any channel should also perform the same way.